src/public_key.rs

```diff
@@ -386,13 +386,18 @@ impl PublicKey {
             PublicKeyRepr::Secp256k1(..) => secp256k1::PUBLIC_KEY_LENGTH,
         }
     }
+
+    #[cfg(feature = "solana")]
+    pub fn to_solana(&self) -> Result<solana_sdk::pubkey::Pubkey> {
```

Comment:
Why have this now? Really not needed?

Reply:
In the wallet, I will need to do this otherwise:
```rust
let solana_key: solana_sdk::pubkey::Pubkey =
                    public_key.clone().try_into()?;
```
Which means I have to solana_sdk into scope whch means I have to import it into the Cargo.toml ... but that's annoying.

With this function, I can just do:
```rust
let solana_key = public_key.to_solana()?;
```

Reply:
Meh not what I’d do since it adds yet another api contract, but it’s no biggie

Reply:
You'd rather just have the client do this after adding solana_sdk to their Cargo.toml?
```rust
let solana_key: solana_sdk::pubkey::Pubkey =
                    public_key.clone().try_into()?;
```
Or is there another way?

Reply:
ok i'll just drop this PR and do that